plugins/inputs/nftables/sample.conf

```diff
@@ -0,0 +1,4 @@
+[[inputs.nftables]]
+# A List of tables to monitor. For the data to be collected, the rule must
+# have a counter and comment declared on it.
+tables = [ "filter" ]
```

Comment:
Missing use_sudo and binary options, Should document all available configuration options